docs/tutorials/tutorials.md

```diff
@@ -0,0 +1,382 @@
+        exists: recommended
+        doc: Datetime of the end of the measurement.
+    ```
+=== "XML"
+    ```xml
+    <group type="NXentry">
+      <field name="definition" type="NX_CHAR">
+        <attribute name="\@version"/>
+        <enumeration>
+          <item value="NXmpes"/>
+        </enumeration>
+      </field>
+      <field name="title"/>
+      <field name="start_time" type="NX_DATE_TIME">
+        <doc>Datetime of the start of the measurement.</doc>
+      </field>
+      <field name="end_time" type="NX_DATE_TIME" recommended="true">
+        <doc>Datetime of the end of the measurement.</doc>
+      </field>
+    </group>
+    ```
+
+#### NeXus Link
+The NeXus `link` reduces data duplication since several concepts of the same kind (e.g., NeXus `group`, `field`, or `attribute`) can refer to a single copy of a data element. In YAML format, NeXus `link` is defined denoting the link inside parenthesis. At the same time, the target concept containing the data must be mentioned under the `target` child.
+
+
+
+**NeXus link in YAML and XML format**
+
+=== "YAML"
+    ```yaml
+    reference_measurement(link):
+      target: /entry
+      doc: A link to a full data collection.
+    ```
+=== "XML"
+    ```xml
+    <link type="NXentry" target="/entry">
+      <doc>A link to a full data collection.</doc>
+    </link>
+    ```
+
+In the YAML example above, `reference_measurement` is defined as a link referring to an instance of the `NXentry` group with its `target` attribute denoting a value `/entry`. This structure ensures that the concept referencing the data is effectively linked to the designated target, thereby reducing the redundancy and maintaining data integrity within the NeXus framework.
+
+
+<!-- ### NeXus Choice
+NeXus `choice` concept is designed to choose a concept from a number of concepts of the same kind (e.g., a NeXus field). The `choice` options allows for defining a scientific concept in several modes for different situations (e.g., for different instrument configurations or measurement modes).
+
+**NeXus choice in YAML format**
+__Not implemented in the `nyaml` tool! Coming soon__
+
+```yaml
+pixel_shape(choice):
+  (NXoff_geometry):
+    doc: Shape description of each pixel. Use only if all pixels in the detector
+      are of uniform shape.
+  (NXcylindrical_geometry):
+    doc: Shape description of each pixel. Use only if all pixels in the detector
+      are of uniform shape and require being described by cylinders.
+```
+
+In this `choice` example, `pixel_shape` could be any of the groups `(NXoff_geometry)` and `(NXcylindrical_geometry)`, depending on the geometry of the pixels. -->
+
+### Special Keywords in YAML
+
+In the YAML schema certain keywords hold self significance beyond their literal interpretations. These special keywords are utilized to elucidate and denote various NeXus terms like `attributes`, `fields`, `links`, and `groups`, thereby enhancing the clarity and specificity of the data representation.
```

Comment:
Certain keywords hold significance beyond ... to elucidate ...

This improves clarity and specificity

Reply:
Addressed!